src/api/favorite/index.ts

```
@@ -0,0 +1,105 @@
+
+  // Supabase のクエリを実行
+  const { data, error } = (await supabase
+    .from("user_favorite")
+    .select("favorite(id, created_at, is_deleted, pokemon_id)")
+    .eq("user_id", user?.id)) as {
+    data: UserFavoriteType[] | null;
+    error: any;
+  };
+
+  if (error) {
+    console.error("Supabase error:", error);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)